irods/message/ordered.py

```diff
@@ -2,24 +2,18 @@
 # http://askawizard.blogspot.com/2008/10/ordered-properties-python-saga-part-5.html
 from itertools import count
 
-try:
-    next_counter = count().__next__
-except AttributeError:
-    next_counter = count().next
-
-
 class OrderedProperty:
 
     def __init__(self, *args, **kws):
-        self._creation_counter = next_counter()
+        self._creation_counter = count().__next__()
```

Comment:
Previously, a global iterator was being instantiated once at import time, and `next` was being called each time on that same iterator. This is now constructing a new iterator each time and counting to 1. I don't think this is equivalent to what was being done before.